stdlib/2and3/codecs.pyi

```diff
@@ -33,8 +34,8 @@ _encoded = bytes
 
```

Comment:
It's really too bad that this file uses all-lowercase names to indicate various type aliases. But maybe now's not the right time to fix that.